t/app/controller/root.t

```diff
@@ -73,4 +74,17 @@ FixMyStreet::override_config {
     };
 };
 
+subtest "check_login_disallowed cobrand hook" => sub {
+    my $cobrand = Test::MockModule->new('FixMyStreet::Cobrand::Default');
+    $cobrand->mock('check_login_disallowed', sub {
+            return 1;
+        }
+    );
+
+    $mech->get('/');
+    is $mech->status, 302, 'disallowed page issues a redirect';
+    is $mech->res->headers->header('location'), 'http://localhost/auth?r=auth', 'redirects to auth page';
```

Comment:
Redirecting to `/auth?r=auth` doesn't feel right; if you fetched the front page, shouldn't it have been `/auth?r=`?